var/spack/repos/builtin/packages/git/package.py

```diff
@@ -145,6 +145,8 @@ class Git(AutotoolsPackage):
             placement='git-manpages',
             when='@{0}'.format(release['version']))
 
+    variant('subversion', default=False, description='Build svn-fe for git svn')
```

Comment:
It looks like `git svn` is already supported without this variant, so IMO the name of this variant could be more specific: `include_svnfe`?